src/dot/commons/pose/head_pose.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import cv2
+import mediapipe as mp
+import numpy as np
+
+mp_face_mesh = mp.solutions.face_mesh
+face_mesh = mp_face_mesh.FaceMesh(
+    min_detection_confidence=0.5, min_tracking_confidence=0.5
+)
+mp_drawing = mp.solutions.drawing_utils
+
+
+def pose_estimation(
+    image: np.array, roll: int = 3, pitch: int = 3, yaw: int = 3
+) -> int:
+    """Given an image and desired roll, pitch and yaw angles, returns whether head pose estimation meets requirements.
+
+    Args:
+        image (np.array): Image to estimate head pose.
+        roll (int, optional): Rotation margin in X axis. Defaults to 3.
+        pitch (int, optional): Rotation margin in Y axis. Defaults to 3.
+        yaw (int, optional): Rotation margin in Z axis. Defaults to 3.
+
+    Returns:
+        int: _description_
+    """
+
+    results = face_mesh.process(image)
+    img_h, img_w, img_c = image.shape
+    face_3d = []
+    face_2d = []
+    if results.multi_face_landmarks:
+        for face_landmarks in results.multi_face_landmarks:
+            for idx, lm in enumerate(face_landmarks.landmark):
+                if (
+                    idx == 33
+                    or idx == 263
+                    or idx == 1
+                    or idx == 61
+                    or idx == 291
+                    or idx == 199
+                ):
```

Comment:
you can define a list of these landmark indices and use `if idx in landmark_list` you can use an appropriate variable name here. I just used a random one as an example.